optimize/util/optimize-commons/src/main/resources/service-config.yaml

```diff
@@ -566,32 +566,6 @@ email:
     # Possible values are 'NONE', 'STARTTLS' or 'SSL/TLS'
     securityProtocol: ${CAMUNDA_OPTIMIZE_EMAIL_AUTHENTICATION_SECURITY_PROTOCOL:NONE}
 
-webhookAlerting:
```

Comment:
Are we doing config removals already? If so I will also add this into my PRs. My personal preference would also be removing on a per feature basis and then adjusting once at the end in terms of any structural changes, but I've been avoiding doing this since from the planning I took we're supposed to do it all in one big config ticket at the end.